Makefile

```diff
@@ -715,7 +715,6 @@ HEADER_FILES = \
   StrictifyFloat.h \
   Substitute.h \
   Target.h \
-  ThreadPool.h \
```

Comment:
Wait... why did you remove this? I thought that the usage of `<thread>` in here is what was unneeded?

Reply:
Ugh.
The story goes as following: i removed `ThreadPool.h`, noticed that breaks build,
unstaged `ThreadPool.h` removal, then fixed s/PRIVATE/PUBLIC/,
staged changes, checked that it still works for me, and pushed.
The key caveat is that i did not notice that the makefile change got restaged :/